build/config.gni

```diff
@@ -10,7 +10,7 @@ declare_args() {
   brave_channel = ""
   is_release_channel = false
   base_sparkle_update_url = ""
-  enable_sparkle = is_official_build && is_mac
+  enable_sparkle = (is_official_build || !is_component_build) && is_mac
```

Comment:
I think it's safe to check `!is_component_build && is_mac`, because https://chromium.googlesource.com/chromium/src/+/main/BUILD.gn#59

Reply:
Fixed